source/_posts/2015-04-13-working-with-init.markdown

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "working with init"
+date: 2015-04-13 10:05:01 -0700
+comments: true
+categories:
+published: false 
+---
+##Create a linux startup service and init.d script from scratch, using chef and debian linux. 
```

Comment:
Maybe add a space after `##`. I'm not sure how well support it is without a space.